mapbox-streets/poi-label.json

```diff
@@ -0,0 +1,462 @@
+                        "geometry_type": "Point",
+                        "data_layer": "poi_label",
+                        "data_layer_fields": {
+                            "filterrank": 1,
+                            "sizerank": 2,
+                            "class": "park_like"
+                        }
+                    },
+                    {
+                        "id": 4013542811,
+                        "geometry_type": "Point",
+                        "data_layer": "poi_label",
+                        "data_layer_fields": {
+                            "filterrank": 1,
+                            "sizerank": 0,
+                            "class": "park_like"
+                        }
+                    },
+                    {
+                        "id": 3588385560,
+                        "geometry_type": "Point",
+                        "data_layer": "poi_label",
+                        "data_layer_fields": {
+                            "filterrank": 1,
+                            "sizerank": 16,
+                            "class": "park_like"
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -77.035,
+                    38.889
+                ]
+            },
+            "properties": {
+                "place_name": "National mall, Washington DC, USA",
+                "zoom": 15,
+                "description": "National mall with smaller POIs inside of it.",
+                "highlights": [
+                    {
+                        "id": 59916094,
+                        "geometry_type": "Point",
+                        "data_layer": "poi_label",
+                        "data_layer_fields": {
+                            "filterrank": 1,
+                            "sizerank": 2,
+                            "class": "park_like"
+                        }
+                    },
+                    {
+                        "id": 33084109630,
+                        "geometry_type": "Point",
+                        "data_layer": "poi_label",
+                        "data_layer_fields": {
+                            "sizerank": 16,
+                            "class": "historic",
+                            "filterrank": 2
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -98.457,
+                    29.358
+                ]
+            },
+            "properties": {
+                "place_name": "San Antonio State Hospital, San Antonio TX, USA",
+                "zoom": 15,
+                "description": "San Antonio State Hospital.",
```

Comment:
The description field is optional – maybe don't include one if it's not going to have any more information that isn't already in the name? Ideally though I think we use this field: https://github.com/mapbox/gazetteer/blob/gazetteer-3/CONTRIBUTING.md#description